src/Data/AudienceFolder.elm

```diff
@@ -24,6 +36,53 @@ type alias AudienceFolder =
 
 
 
+-- View
+{- | View for audienceFolder serves to simply visualize folder elements. This function could be located in Main, but for better code readability,
+   it belongs in this file. Obtains bool value that indicates whether folder is currently opened or not.
+-}
+
+
+view : AudienceFolder -> Bool -> msg -> msg -> Html msg
+view folder opened select goUp =
+    div
+        [ class "folder"
+        , if opened then
+            onClick goUp
+
+          else
+            onClick select
+        ]
+        [ if opened then
+            span [ style "float" "left", style "margin-right" "10px" ]
+                [ Outlined.drive_file_move_rtl 20 (Color <| Color.rgb 96 181 204)
```

Comment:
This was a bit suspicious behaviour from a UX point of view (IMHO), I would rather see a `Return ◀️` button than the same folder name with a very little icon inside the folder which is almost unnoticeable (very minor though)